webview/src/experiments/components/App.test.tsx

```diff
@@ -182,12 +183,7 @@ describe('App', () => {
     const headerB = screen.getByText('B')
     const headerD = screen.getByText('D')
 
-    dragAndDrop(
-      headerB,
-      // eslint-disable-next-line testing-library/no-node-access
-      headerD.parentElement?.parentElement || headerD,
-      DragEnterDirection.AUTO
-    )
+    dragAndDrop(headerB, getParentElement(headerD, 2), DragEnterDirection.AUTO)
```

Comment:
I think a [testing-library custom query](https://testing-library.com/docs/dom-testing-library/api-custom-queries/) might be a better level of abstraction for this, both in ease of use and the ability to easily adapt to changes in the experiments table's DOM in the future.

Reply:
Thanks for the suggestion! I'll change this.